tools/benchmark/src/ReporterUtilities.ts

```diff
@@ -76,3 +77,48 @@ export function geometricMean(values: number[]): number {
     }
     return Math.exp(sum / values.length);
 }
+
+/**
+ * T-Distribution two-tailed critical values for 95% confidence.
+ * For more info see http://www.itl.nist.gov/div898/handbook/eda/section3/eda3672.htm.
+ */
+/* eslint-disable quote-props,key-spacing,no-multi-spaces */
+const tTable = {
+    "1":  12.706, "2":  4.303, "3":  3.182, "4":  2.776, "5":  2.571, "6":  2.447,
+    "7":  2.365,  "8":  2.306, "9":  2.262, "10": 2.228, "11": 2.201, "12": 2.179,
+    "13": 2.16,   "14": 2.145, "15": 2.131, "16": 2.12,  "17": 2.11,  "18": 2.101,
+    "19": 2.093,  "20": 2.086, "21": 2.08,  "22": 2.074, "23": 2.069, "24": 2.064,
+    "25": 2.06,   "26": 2.056, "27": 2.052, "28": 2.048, "29": 2.045, "30": 2.042,
+    "infinity": 1.96,
+};
+/* eslint-enable */
+
+/**
+ * Compute statistics for an array of numbers. For homogeneity, it outputs the same
+ * object that the Benchmark library does.
+ *
+ * @param array - List of numbers for which to compute the statistics.
+ * @returns
+ */
+export function getArrayStatistics(array: number[]): Benchmark.Stats {
+    const n = array.length;
+    let max = -Infinity;
+    let min = Infinity;
+    let mean = 0;
+    array.forEach((x) => {
+        mean += x;
+        if (x > max) { max = x; }
+        if (x < min) { min = x; }
+    });
+    mean /= n;
+
+    const variance = array.map((x) => (x - mean) ** 2).reduce((a, b) => a + b) / n;
+    const deviation = Math.sqrt(variance);
+    const sem = deviation / Math.sqrt(n);
+    const df = n - 1;
+    const critical = tTable[Math.round(df) || "1"] ?? tTable.infinity;
```

Comment:
Math.round(df) should be a number, so the index here doesn't look right.

Reply:
Yeah, I'll toString() it (and remove the redundant `round()` since it's already an integer) so it's clearer that this is a property access on an object, not an array; but this actually works anyway because JavaScript 🤷 :

![image](https://user-images.githubusercontent.com/716334/181806897-a51b5314-86fa-43f6-a69d-fc686e5b7c61.png)

Reply:
ah, yeah duh.